app/controllers/close_contacts_controller.rb

```diff
@@ -54,4 +54,22 @@ def update
                                created_by: current_user.email,
                                comment: "User edited a close contact (ID: #{cc.id}).")
   end
+
+  # Delete an existing close contact record
+  def destroy
+    redirect_to root_url && return unless current_user.can_create_patient_close_contacts?
+    cc = current_user.get_patient(params.permit(:patient_id)[:patient_id])&.close_contacts&.find_by(id: params.permit(:id)[:id])
+    redirect_to root_url && return if cc.nil?
+    cc.destroy
+    if cc.destroyed?
+      reason = params.permit(:delete_reason)[:delete_reason]
+      History.close_contact_edit(patient: params.permit(:patient_id)[:patient_id],
+                              created_by: current_user.email,
```

Comment:
don't forget to resolve the rubocop issues: https://github.com/SaraAlert/SaraAlert/pull/842/checks?check_run_id=2603934392#step:16:17

Reply:
@timwongj I've fixed the issues on my code, but there are some rubocop errors on code I haven't touched and isn't relevant to this pr, and I just want a second opinion (from you would count) before including that in this pr rather than making the changes elsewhere.

Reply:
if you rebase those should be fixed